public/styles/bestPractiseForStyledComponents.js

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+
+/**
+ *
+ * @summary This component can be used independently or be the basis of other component
```

Comment:
Can you add all examples to `README.md` in this folder and link this file to main `README.md`?